routes/queryGraphQL.js

```diff
@@ -0,0 +1,95 @@
+const { request, gql } = require('graphql-request')
```

Comment:
And this file should probably be out of routes folder - just to make code reading easier in the future

Reply:
Yes, we should refactor the structure here. Good catch

Reply:
Will change in a future refactoring.